src/swap/defi/thorchain.ts

```diff
@@ -195,9 +195,12 @@ export const asExchangeInfo = asObject({
   swap: asObject({
     plugins: asObject({
       thorchain: asObject({
-        perAssetSpread: asArray(asAssetSpread),
-        volatilitySpread: asNumber,
-        likeKindVolatilitySpread: asNumber,
+        // perAssetSpread: asArray(asAssetSpread),
+        perAssetSpreadStreaming: asOptional(asArray(asAssetSpread)),
+        // volatilitySpread: asNumber,
+        volatilitySpreadStreaming: asOptional(asNumber),
+        // likeKindVolatilitySpread: asNumber,
+        likeKindVolatilitySpreadStreaming: asOptional(asNumber),
```

Comment:
Instead of keeping comments, we can just rely on our repository history for access to the code prior to the change. Unless these are still valid for the info server; then why not leave them with a deprecated comment?

Reply:
Crud. I just realize we still need to apply the non-streaming spreads in case we get a non-streaming quote from the api. another change incoming.